plugin/kubernetes/controller.go

```diff
@@ -191,6 +200,17 @@ func newdnsController(ctx context.Context, kubeClient kubernetes.Interface, opts
 		defaultResyncPeriod,
 		cache.ResourceEventHandlerFuncs{})
 
+	dns.nodeLister, dns.nodeController = object.NewIndexerInformer(
```

Comment:
if `k8s_node` is not enabled, we should not init/run the node informer (since without `k8s_node` the informer would be consuming resources for no purpose)